package.json

```diff
@@ -20,15 +20,16 @@
   },
   "homepage": "https://github.com/QuickBlox/javascript-media-recorder#readme",
   "devDependencies": {
-    "browserify": "^13.1.1",
+    "browserify": "^13.3.0",
     "docdash": "^0.4.0",
     "gulp": "^3.9.1",
     "gulp-connect": "^5.0.0",
     "gulp-notify": "^2.2.0",
     "gulp-plumber": "^1.1.0",
     "gulp-uglify": "^2.0.0",
     "jsdoc": "^3.4.3",
-    "vinyl-source-stream": "^1.1.0"
+    "vinyl-source-stream": "^1.1.0",
+    "lamejs": "^1.1.0"
```

Comment:
Is lamejs dependency for QBMediaRecorder? 
This is not required. Better use CDN in sample